# Adding in a patch to fix the default x terminal
# Author: Paul Tagliamonte <paultag@ubuntu.com>
# Applied-Upstream: 1.1.1+git20100807.0cc08f9-1
# Last-Update: 2010-08-12
diff --git a/data/keys b/data/keys
index 7e2557c..b02f80f 100644
--- a/data/keys
+++ b/data/keys
@@ -58,7 +58,7 @@ Mod4 8 :Tab 8
 Mod4 9 :Tab 9
 
 # open a terminal
-Mod1 F1 :Exec xterm
+Mod1 F1 :Exec x-terminal-emulator
 
 # open a dialog to run programs
 Mod1 F2 :Exec fbrun
